scim-spec/scim-spec-protocol/src/main/java/edu/psu/swe/scim/spec/protocol/data/PatchOperationPathAdapter.java

```diff
@@ -28,7 +28,7 @@ public PatchOperationPath unmarshal(String v) throws Exception {
     if (v == null) {
       return null;
     }
-    return new PatchOperationPath();
+    return new PatchOperationPath(v);
```

Comment:
I wanted to write an integration test for this based off the spec sample PATCH operations, but there doesn't appear to be a source for an Object Mapper in the Protocol sub-project.